packages/browser/src/login/oidc/redirectHandler/AuthCodeRedirectHandler.ts

```diff
@@ -146,44 +147,44 @@ export class AuthCodeRedirectHandler implements IRedirectHandler {
         errorIfNull: true,
       }
     )) as string;
-    const isDpop =
-      (await this.storageUtility.getForUser(storedSessionId, "dpop")) ===
-      "true";
 
-    // Since we throw if not found, the type assertion is okay
-    const issuer = (await this.storageUtility.getForUser(
+    const {
+      issuerConfig,
+      codeVerifier,
+      redirectUrl: storedRedirectIri,
+      dpop: isDpop,
+    } = await loadOidcContextFromStorage(
       storedSessionId,
-      "issuer",
-      { errorIfNull: true }
-    )) as string;
+      this.storageUtility,
+      this.issuerConfigFetcher
+    );
 
-    // Store the current session ID specifically in 'localStorage' (i.e., not using
-    // any other storage mechanism), as we don't deem this information to be
-    // sensitive, and we want to ensure it survives a browser tab refresh.
-    window.localStorage.setItem(KEY_CURRENT_SESSION, storedSessionId);
+    if (codeVerifier === undefined) {
+      throw new Error(
```

Comment:
Will these be caught by the SDK?

Reply:
They should, but I think it would be cleaner to dedicate a complete PR to error management, because that's not the only error that should be caught at a higher level.

Reply:
Yeah, agree there, I just wanted to check as I wasn't sure